packages/jupyter-ai/pyproject.toml

```diff
@@ -24,7 +24,7 @@ classifiers = [
 dependencies = [
     "jupyter_server>=1.6,<3",
     "jupyterlab~=4.0",
-    "openai~=0.26",
+    "openai~=1.6.1",
```

Comment:
We shouldn't have a hard dependency on `openai`, and the declaration in the optional `jupyter_ai_magics[all]` group needs a version range specifier. You should remove `openai` here and instead declare this under `jupyter_ai_magics[all]`.